src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/Tensor_1.cs

```diff
@@ -125,13 +125,13 @@ private Tensor()
         /// <inheritdoc cref="TensorSpan{T}.this[ReadOnlySpan{nint}]" />
         public ref T this[params scoped ReadOnlySpan<nint> indexes]
         {
-            get => ref Unsafe.Add(ref MemoryMarshal.GetArrayDataReference(_values), _shape.GetLinearOffset<TensorShape.GetOffsetAndLengthForNInt, nint>(indexes));
+            get => ref Unsafe.Add(ref Unsafe.Add(ref MemoryMarshal.GetArrayDataReference(_values), _start), _shape.GetLinearOffset<TensorShape.GetOffsetAndLengthForNInt, nint>(indexes));
```

Comment:
This could've been `ref Unsafe.Add(ref MemoryMarshal.GetArrayDataReference(_values), _start + _shape.GetLinearOffset<TensorShape.GetOffsetAndLengthForNInt, nint>(indexes));`

I've updated it locally